synbiohub_adapter/upload_sbol/upload_sbol.py

```diff
@@ -355,15 +386,18 @@ def __merge_remote_sub_collections(self, docs, collection_namespace):
                 remote_uri = '/'.join([collection_namespace, sub_collection.displayId, sub_collection.version])
 
                 try:
-                    self.part_shop.pull(remote_uri, remote_doc, False)
+                    if self.spoofed_url:
+                        self.part_shop.pull(remote_uri.replace(self.spoofed_url, self.url), remote_doc, False)
+                    else:
+                        self.part_shop.pull(remote_uri, remote_doc, False)
 
                     try:
-                        remote_sub_collection = remote_doc.getCollection(remote_uri.replace(self.url + '/', ''))
+                        remote_sub_collection = remote_doc.getCollection(remote_uri)
 
                         sub_collection.members = sub_collection.members + remote_sub_collection.members
-                    except:
+                    except RuntimeError:
                         raise SubCollectionMergeError(sub_collection.displayId, sub_collection.version)
-                except:
+                except LookupError:
```

Comment:
What does a LookupError indicate in this context? And why is it ok to ignore? Could a comment be added here to let a future programmer know what the reasoning is?

Reply:
Adding comment to indicate that if sub-Collection does not already exist on SynBioHub, then there is nothing to merge and passing is okay.